### PR TITLE
Feature/memos

### PR DESCRIPTION
**Pull request by phillipgalbraith**
## Description

Notes are rebranded as Memos in the UI.   Users should be able to recognize the note-sharing feature as a communication tool.  Rebranding Notes as Memos follows participants' failure to find a place to post prioritized or time-sensitive communications to groups of admin or groups of mentors in usability tests.

 A Memos link is added in the Navbar with a dropdown to emphasize the regular use of Memos similar to a notifications feature.

Co-authors:
Areccus Branch
Prashish Bhandari
Brittni Karat

#### Video Link

[Loom Video](https://www.loom.com/share/4e4b28eaff0f4627a855bcbba10a80bc)

#### Trello Link
<blockquote class="trello-card"><a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;nCqby9zb&#x2F;303-bug-re-branding-notes-feature-based-on-user-flow-feedback-to-memos">Bug - re-branding &quot;Notes&quot; feature based on user flow feedback to &quot;memos&quot;</a></blockquote><script src="https://p.trellocdn.com/embed.min.js"></script>

<blockquote class="trello-card"><a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;KYfnq8hS&#x2F;305-as-a-user-i-would-like-to-be-able-to-access-my-memos-quickly-so-that-i-can-add-edit-create-them-as-needed">As a user, I would like to be able to access my memos quickly, so that I can add&#x2F;edit&#x2F;create them as needed</a></blockquote><script src="https://p.trellocdn.com/embed.min.js"></script>

## Type of change

- [X] New feature (non-breaking change which adds functionality)

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/Navbar/Navbar.js [hunk 1]
```diff
@@ -62,34 +77,35 @@ const Navbar = ({ isAuthenticated, userProfile }) => {
...
+                      <div className="userInfo">
+                        <div className="username">
+                          Welcome {user.first_name}
+                        </div>
+                      </div>
                     </div>
-                  </div>
-                </Dropdown>
+                  </Dropdown>
+                </div>
```
Comment (maidiance) [inline src/components/pages/Navbar/Navbar.js:108]: Why is this in the opposite order?

**Review by maidiance — APPROVED** [review]
Looks like a necessary and clean rebrand. Just curious about two lines switching order in particular.

**Review by BrittniKarat — APPROVED** [review]
I can't find any improvements in this code. The commits are labeled well. The PR is small and focused. The changes will absolutely hep the user flow. Great work.

File: src/components/pages/Navbar/Navbar.js (hunk 1 above)
Comment (phillipgalbraith) [reply src/components/pages/Navbar/Navbar.js]: I added a div that wraps two dropdowns.

**Review by acoman-cloud — APPROVED** [review]
Great choice to change the name of notes to memos. Great changes to ui and ux.

**Review by beefybroccoli — APPROVED** [review]
The scope is sufficiently small. The UI could use more functionalities. This is a great small change toward a bigger improvement.

**Pull request merged**